src/Storageless/Session/Zend/SessionAdapter.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+declare(strict_types=1);
+
+namespace PSR7Sessions\Storageless\Session\Zend;
+
+use PSR7Sessions\Storageless\Session\SessionInterface;
+use Zend\Expressive\Session\SessionInterface as ZendSessionInterface;
+use function sprintf;
+
+final class SessionAdapter implements ZendSessionInterface
+{
+    /** @var SessionInterface */
+    private $session;
+
+    public function __construct(SessionInterface $session)
+    {
+        $this->session = $session;
+    }
+
+    /**
+     * @return array
```

Comment:
This needs to be more specific. If `mixed[]` is the best we got, that's enough.

Reply:
For instance, `mixed[]` also allows for resources, but we only allow for json-serializable data in this package. A restriction is needed

Reply:
Which means something like this? `bool[]|int[]|float[]|string[]|array[]|\stdClass[]`

Reply:
Or better:

`array<int|string, bool|int|float|array<mixed>|\stdClass>`

Reply:
it seems vendor/bin/phpcs does not (yet) support this syntax

Reply:
https://github.com/psr7-sessions/storageless/pull/82#discussion_r275132441

Reply:
Still failing

`@return annotation of method \PSR7Sessions\Storageless\Session\Zend\SessionAdapter::toArray() does not specify type hint for items of its traversable return value.`